packages/workbox-core/src/models/messages/messages.ts

```diff
@@ -6,12 +6,14 @@
   https://opensource.org/licenses/MIT.
 */
 
-import {MapLikeObject} from '../../types.js';
 import '../../_version.js';
 
 
+interface LoggableObject {
+  [key: string]: string | number;
```

Comment:
Do you know which named error function returns a `number`? I had thought all of them returned `string`, and if one doesn't, that might be a bug.

Reply:
Gotcha.